spec/controllers/address_controller_spec.rb

```diff
@@ -79,7 +77,8 @@
   end
 
   def current_app
-    @_current_app ||= SnapApplication.create!(
+    @_current_app ||= FactoryGirl.create(
```

Comment:
yes we for sure should! they were not using FG when we first started, so we added it to work alongside their custom factory thingy. Could not use FG helper methods here because there would be clashing namespaces. will add soon!